.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,14 @@
 blank_issues_enabled: false
+contact_links:
+  - name: 🤔 Questions or ideas?
+    url: https://github.com/airjp73/remix-validated-form/discussions
+    about: Ask long-form questions and discuss ideas.
+  - name: 💬 Remix Discord / Community Chat
+    url: https://rmx.as/discord
+    about: Interact with other people using Remix 📀
+  - name: 💬 New Updates (Twitter)
```

Comment:
I love the `🤔 Questions or ideas?` link -- I had no idea this `contact_links` config existed. Could we remove the Twitter and YouTube links though? I frequent the Discord channel so I'm okay with directing people there with general remix questions.

Reply:
Sure I can remove! 
The one benefit of having it as a maintainer is that it empowers people to dig in a bit further before they create an issue & take some of your mental bandwidth away - you know in case you're watching Netflix or relaxing lol

Removed! 🙏